CMakeLists.txt

```diff
@@ -39,6 +39,8 @@ endif()
 
 include( ProjectSettings.cmake )
 
+include( FrameworkInternals/BoostSetup.cmake )
```

Comment:
As discussed in today's meeting, to handle situations where `find_package(Boost ...)` does not work so well, I would like to insert my own boost locator file. So I propose
```
if( NOT DEFINED BOOST_LOCATOR_CMAKE_FILE)
  set( BOOST_LOCATOR_CMAKE_FILE FrameworkInternals/BoostSetup.cmake )
endif()
message( STATUS "locating boost using cmake file [${BOOST_LOCATOR_CMAKE_FILE}]" )
include( $BOOST_LOCATOR_CMAKE_FILE )

Reply:
So in my build configs I would 
```
set( BOOST_LOCATOR_CMAKE_FILE FindBoostWithBen.cmake )
```
And in those files I must
1. add boost include directories with `include_directories`
2. set variable `BOOST_LIBS`

Reply:
Ok, thanks for the proposal. I will change the var name slightly since it is not only about the location but also the libs needed by quasar.